test/swingsetTests/gallery/test-gallery.js

```diff
@@ -273,3 +273,37 @@ test('gallery - create childPayment, spend, revoke without SES', async t => {
   t.deepEquals(dump.log, expectedSpendAndRevoke);
   t.end();
 });
+
+const expectedGetAllPixels = [
```

Comment:
It's beyond the scope of this PR but we need a stronger unit test approach in general. These should be assertions in the tests.

Reply:
I think you're right that the swingSetTests are inadequate. We have a separate unitTests folder that does allow us to have assertions in the tests. Those tests are included in this PR.